### PR TITLE
[i18n-ar] Translated file : docs/source/ar/tasks/sequence_classification.md into Arabic

### PR DESCRIPTION
**Pull request by AhmedAlmaghz**
## What does this PR do?
Translated the `docs/source/ar/tasks/sequence_classification.md` file of the documentation to `Arabic العربية`. I use transformers for my side-project and wanted to contribute. As a beginner, this seemed appropriate.

### Some questions:

- I can't access the pr documentation endpoint for `Arabic العربية`. Is there a missing step?
- I left all the other files alone after copying the English docs. Is this acceptable for future updates or should I add contents to the `_toctree.yml` only when translation is complete? Should I delete all other files for a cleaner review?
- In `Arabic`, people use both `Transformer` and `المحولات` to describe the model. However, as this is more of a product of HuggingFace I opted not to translate it. May you please let me know which you would prefer?

Thank you in advance for your review.

Part of #32435

### Before submitting

- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests), Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/) ?Please add a link to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

 ### Who can review?
 @stevhliu , @MKhalusova , @abodacs, may you please review this PR?

### CODE REVIEW

**Comment by AhmedAlmaghz** [comment]
مرحباً أخي عبالله @abodacs 
هذه قائمة المهام التالية للترجمة والمراجعة

# TASK GUIDES

## NATURAL LANGUAGE PROCESSING
- [ ]  Text classification #35192
- [ ]  Token classification
- [ ]  Question answering
- [ ]  Causal language modeling
- [ ]  Masked language modeling
- [ ]  Translation
- [ ]  Summarization 
- [ ]  Multiple choice

## AUDIO
- [ ]  Audio classification
- [ ]  Automatic speech recognition

## COMPUTER VISION
- [ ]  Image classification
- [ ]  Image segmentation
- [ ]  Video classification
- [ ]  Object detection
- [ ]  Zero-shot object detection
- [ ]  Zero-shot image classification
- [ ]  Depth estimation
- [ ]  Image-to-Image
- [ ]  Image Feature Extraction
- [ ]  Mask Generation
- [ ]  Keypoint Detection
- [ ]  Knowledge Distillation for Computer Vision

## Multimodal
- [ ]  Image captioning
- [ ]  Document Question Answering
- [ ]  Visual Question Answering
- [ ]  Text to speech
- [ ]  Image-text-to-text
- [ ]  Video-text-to-text

## Generation
- [ ]  Customize the generation strategy
- [ ]  Best Practices for Generation with Cache

## Prompting
- [ ]  Image tasks with IDEFICS
- [ ]  LLM prompting guide


أشكرك على جهودك الرائعة

**Comment by AhmedAlmaghz** [comment]
> السلام عليكم هل من الممكن ربط المهام برقمها؟ وأيضا التأكد من عدم وجود مهام لم تترجم للعربية؟

وعليكم السلام ورحمة الله وبركاته 

سنعمل على مجموعة مجموعة
سنبداء ب : NATURAL LANGUAGE PROCESSING

ثم الذي تليها وهكذا

تحياتي وتقديري أخي عبدالله @abodacs

**Comment by AhmedAlmaghz** [comment]
مرحبا أخي عبدالله @abodacs 
كيف حالك ؟ هل أنت مشغول ؟
لماذا لم تتم المراجعة ؟

تحياتي وتقديري لشخصكم الكريم

File: docs/source/ar/tasks/sequence_classification.md [hunk 1]
```diff
@@ -0,0 +1,392 @@
...
+
+غالبًا ما يكون تضمين مقياس أثناء التدريب مفيدًا لتقييم أداء نموذجك. يمكنك تحميل طريقة تقييم بسرعة باستخدام مكتبة 🤗 [Evaluate](https://huggingface.co/docs/evaluate/index) . بالنسبة لهذه المهمة، قم بتحميل مقياس [الدقة](https://huggingface.co/spaces/evaluate-metric/accuracy) (راجع جولة 🤗 Evaluate [السريعة](https://huggingface.co/docs/evaluate/a_quick_tour) لمعرفة المزيد حول كيفية تحميل وحساب مقياس):
+
+```py
+>>> import evaluate
+
+>>> accuracy = evaluate.load("accuracy")
+```
+
+ثم قم بإنشاء وظيفة تقوم بتمرير تنبؤاتك وتصنيفاتك إلى [`~evaluate.EvaluationModule.compute`] لحساب الدقة:
```
Comment (abodacs) [inline docs/source/ar/tasks/sequence_classification.md:129]:
```suggestion
ثم أنشئ دالة تقوم بتمرير تنبؤاتك وتصنيفاتك إلى [`~evaluate.EvaluationModule.compute`] لحساب الدقة:
```

File: docs/source/ar/tasks/sequence_classification.md [hunk 2]
```diff
@@ -0,0 +1,392 @@
...
+```py
+>>> import numpy as np
+
+>>> def compute_metrics(eval_pred):
+...     predictions, labels = eval_pred
+...     predictions = np.argmax(predictions, axis=1)
+...     return accuracy.compute(predictions=predictions, references=labels)
+```
+
+وظيفتك `compute_metrics` جاهزة الآن، وستعود إليها عندما تقوم بإعداد تدريبك.
```
Comment (abodacs) [inline docs/source/ar/tasks/sequence_classification.md:140]:
```suggestion
دالة `compute_metrics` جاهزة الآن، وستعود إليها عند إعداد التدريب.
```

File: docs/source/ar/tasks/sequence_classification.md [hunk 3]
```diff
@@ -0,0 +1,392 @@
...
+>>> model = AutoModelForSequenceClassification.from_pretrained(
+...     "distilbert/distilbert-base-uncased", num_labels=2, id2label=id2label, label2id=label2id
+... )
+```
+
+في هذه المرحلة، هناك ثلاث خطوات فقط متبقية:
+
+1. حدد معلمات التدريب الخاصة بك في [`TrainingArguments`]. المعلمة الوحيدة المطلوبة هي `output_dir` والتي تحدد مكان حفظ النموذج الخاص بك. سوف تقوم بدفع هذا النموذج إلى Hub عن طريق تعيين `push_to_hub=True` (يجب أن تكون مسجلاً الدخول إلى Hugging Face لتحميل نموذجك). في نهاية كل حقبة، سيقوم [`Trainer`] بتقييم الدقة وحفظ نقطة تفتيش التدريب.
+2. قم بتمرير معاملات التدريب إلى [`Trainer`] جنبًا إلى جنب مع النموذج، ومجموعة البيانات، والمحلل اللغوي، ومجمع البيانات، ووظيفة `compute_metrics`.
+3. قم باستدعاء [`~Trainer.train`] لضبط نموذجك الدقيق.
```
Comment (abodacs) [inline docs/source/ar/tasks/sequence_classification.md:173]:
```suggestion
1.  حدد مُعامِلات التدريب في [`TrainingArguments`]. المُعامل المطلوب الوحيد هو `output_dir`، لتحديد مكان حفظ النموذج. يمكنك رفع النموذج إلى Hub بتعيين `push_to_hub=True` (يجب تسجيل الدخول إلى Hugging Face لرفع النموذج). سيقوم `Trainer` بتقييم الدقة وحفظ نقاط التحقق في نهاية كل حقبة.
2.  مرر مُعامِلات التدريب إلى `Trainer` مع النموذج، ومجموعة البيانات، والمحلل اللغوي، ومُجمِّع البيانات، ووظيفة `compute_metrics`.
3.  استدعِ [`~Trainer.train`] لضبط النموذج.
```

File: docs/source/ar/tasks/sequence_classification.md [hunk 4]
```diff
@@ -0,0 +1,392 @@
...
+...     data_collator=data_collator,
+...     compute_metrics=compute_metrics,
+... )
+
+>>> trainer.train()
+```
+
+<Tip>
+
+يطبق [`Trainer`] الحشو الديناميكي بشكل افتراضي عند تمرير `tokenizer` إليه. في هذه الحالة، لا تحتاج إلى تحديد مجمع بيانات بشكل صريح.
```
Comment (abodacs) [inline docs/source/ar/tasks/sequence_classification.md:204]:
```suggestion
يستخدم [`Trainer`] الحشو الديناميكي افتراضيًا عند تمرير `tokenizer` إليه. في هذه الحالة،  لا تحتاج لتحديد مُجمِّع البيانات صراحةً. 
```

File: docs/source/ar/tasks/sequence_classification.md [hunk 5]
```diff
@@ -0,0 +1,392 @@
...
+
+</Tip>
+
+بمجرد اكتمال التدريب، شارك نموذجك على Hub باستخدام طريقة [`~transformers.Trainer.push_to_hub`] حتى يتمكن الجميع من استخدام نموذجك:
```
Comment (abodacs) [inline docs/source/ar/tasks/sequence_classification.md:208]:
```suggestion
بعد اكتمال التدريب، شارك نموذجك على Hub باستخدام الطريقة [`~transformers.Trainer.push_to_hub`] ليستخدمه الجميع:
```

File: docs/source/ar/tasks/sequence_classification.md [hunk 6]
```diff
@@ -0,0 +1,392 @@
...
+>>> import tensorflow as tf
+
+>>> batch_size = 16
+>>> num_epochs = 5
+>>> batches_per_epoch = len(tokenized_imdb["train"]) // batch_size
+>>> total_train_steps = int(batches_per_epoch * num_epochs)
+>>> optimizer, schedule = create_optimizer(init_lr=2e-5, num_warmup_steps=0, num_train_steps=total_train_steps)
+```
+
+ثم يمكنك تحميل DistilBERT مع [`TFAutoModelForSequenceClassification`] جنبًا إلى جنب مع عدد التصنيفات المتوقعة، وخرائط التصنيف:
```
Comment (abodacs) [inline docs/source/ar/tasks/sequence_classification.md:233]:
```suggestion
ثم يمكنك تحميل DistilBERT مع [`TFAutoModelForSequenceClassification`] بالإضافة إلى عدد التصنيفات المتوقعة، وتعيينات التسميات:
```

File: docs/source/ar/tasks/sequence_classification.md [hunk 7]
```diff
@@ -0,0 +1,392 @@
...
+```py
+>>> from transformers.keras_callbacks import PushToHubCallback
+
+>>> push_to_hub_callback = PushToHubCallback(
+...     output_dir="my_awesome_model",
+...     tokenizer=tokenizer,
+... )
+```
+
+ثم اربط استدعاءاتك معًا:
```
Comment (abodacs) [inline docs/source/ar/tasks/sequence_classification.md:290]:
```suggestion
ثم اجمع الاستدعاءات معًا:
```

File: docs/source/ar/tasks/sequence_classification.md [hunk 8]
```diff
@@ -0,0 +1,392 @@
...
+
+رائع، الآن بعد أن قمت بضبط نموذج، يمكنك استخدامه للاستدلال!
+
+احصل على بعض النصوص التي ترغب في إجراء الاستدلال عليها:
+
+```py
+>>> text = "This was a masterpiece. Not completely faithful to the books, but enthralling from beginning to end. Might be my favorite of the three."
+```
+
+أسهل طريقة لتجربة نموذجك المضبوط للاستدلال هي استخدامه في [`pipeline`]. قم بتنفيذ `pipeline` لتحليل المشاعر مع نموذجك، ومرر نصك إليه:
```
Comment (abodacs) [inline docs/source/ar/tasks/sequence_classification.md:324]:
```suggestion
أسهل طريقة لتجربة النموذج المضبوط للاستدلال هي استخدامه ضمن [`pipeline`]. قم بإنشاء `pipeline` لتحليل المشاعر مع نموذجك، ومرر نصك إليه:
```

File: docs/source/ar/tasks/sequence_classification.md [hunk 9]
```diff
@@ -0,0 +1,392 @@
...
+قم يتجزئة النص وإرجاع تنسورات PyTorch:
+
+```py
+>>> from transformers import AutoTokenizer
+
+>>> tokenizer = AutoTokenizer.from_pretrained("stevhliu/my_awesome_model")
+>>> inputs = tokenizer(text, return_tensors="pt")
+```
+
+قم بتمرير مدخلاتك إلى النموذج وإرجاع `logits`:
```
Comment (abodacs) [inline docs/source/ar/tasks/sequence_classification.md:347]:
```suggestion
مرر المدخلات إلى النموذج واسترجع `logits`:
```

File: docs/source/ar/tasks/sequence_classification.md [hunk 10]
```diff
@@ -0,0 +1,392 @@
...
+
+```py
+>>> from transformers import AutoModelForSequenceClassification
+
+>>> model = AutoModelForSequenceClassification.from_pretrained("stevhliu/my_awesome_model")
+>>> with torch.no_grad():
+...     logits = model(**inputs).logits
+```
+
+احصل على الفئة ذات أعلى احتمال، واستخدم خريطة `id2label` الخاصة بالنموذج  لتحويلها إلى تصنيف نصي:
```
Comment (abodacs) [inline docs/source/ar/tasks/sequence_classification.md:357]:
```suggestion
استخرج الفئة ذات الاحتمالية الأعلى، واستخدم `id2label` لتحويلها إلى تصنيف نصي:
```

File: docs/source/ar/tasks/sequence_classification.md [hunk 11]
```diff
@@ -0,0 +1,392 @@
...
+قم بتمرير مدخلاتك إلى النموذج وإرجاع `logits`:
+
+```py
+>>> from transformers import TFAutoModelForSequenceClassification
+
+>>> model = TFAutoModelForSequenceClassification.from_pretrained("stevhliu/my_awesome_model")
+>>> logits = model(**inputs).logits
+```
+
+احصل على الفئة ذات أعلى احتمال، واستخدم خريطة `id2label` الخاصة بالنموذج لتحويلها إلى تصنيف نصي:
```
Comment (abodacs) [inline docs/source/ar/tasks/sequence_classification.md:384]:
```suggestion
استخرج الفئة ذات الاحتمالية الأعلى، واستخدم `id2label` لتحويلها إلى تصنيف نصي:
```

**Review by abodacs — CHANGES_REQUESTED** [review]
السﻻم عليكم
تمت المراجعة 
@AhmedAlmaghz 
جزاك الله خيرا
